src/webhooks/components/WebhooksDetailsPage/WebhooksDetailsPage.tsx

```diff
@@ -0,0 +1,128 @@
+import AppHeader from "@saleor/components/AppHeader";
+import { ConfirmButtonTransitionState } from "@saleor/components/ConfirmButton";
+import Container from "@saleor/components/Container";
+import Form from "@saleor/components/Form";
+import FormSpacer from "@saleor/components/FormSpacer";
+import Grid from "@saleor/components/Grid";
+import PageHeader from "@saleor/components/PageHeader";
+import SaveButtonBar from "@saleor/components/SaveButtonBar";
+import { sectionNames } from "@saleor/intl";
+import { maybe } from "@saleor/misc";
+import { UserError } from "@saleor/types";
+import { WebhookEventTypeEnum } from "@saleor/types/globalTypes";
+import React from "react";
+import { useIntl } from "react-intl";
+import { ServiceList_serviceAccounts_edges_node } from "../../types/ServiceList";
+import { Webhook_webhook } from "../../types/Webhook";
+import WebhookEvents from "../WebhookEvents";
+import WebhookInfo from "../WebhookInfo";
+import WebhookStatus from "../WebhookStatus";
+
+export interface FormData {
+  id: string;
+  events: WebhookEventTypeEnum[];
+  isActive: boolean;
+  name: string;
+  secretKey: string | null;
+  targetUrl: string;
+  serviceAccount: string;
+  allEvents: boolean;
+}
+
+export interface WebhooksDetailsPageProps {
+  disabled: boolean;
+  errors: UserError[];
+  webhook: Webhook_webhook;
+  services: ServiceList_serviceAccounts_edges_node[];
+  saveButtonBarState: ConfirmButtonTransitionState;
+  onBack: () => void;
+  onDelete: () => void;
+  onSubmit: (data: FormData) => void;
+}
+
+const WebhooksDetailsPage: React.StatelessComponent<
+  WebhooksDetailsPageProps
+> = ({
+  disabled,
+  errors,
+  webhook,
+  saveButtonBarState,
+  services,
+  onBack,
+  onDelete,
+  onSubmit
+}) => {
+  const intl = useIntl();
+  const initialForm: FormData = {
+    allEvents: maybe(
+      () =>
+        maybe(() => webhook.events, [])[0].eventType ===
```

Comment:
I'm not 100% sure but it doesn't have to be first element in array. `!!webhooks.events.find(event => event.eventType === WebhookEventTypeEnum.ALL_EVENTS)` would be safer